geode-core/src/main/java/org/apache/geode/Delta.java

```diff
@@ -55,7 +55,20 @@
   void fromDelta(DataInput in) throws IOException, InvalidDeltaException;
 
   /**
-   * Allows Delta implementations to ensure bucket sizes are recalculated after delta is applied
+   * By default, buckets do not recalculate their size when deltas are applied. This optimizes for
+   * the case where the sie of an entry does not change. However, if the size does increase or
+   * decrease, this default behavior can result in the memory usage statistics becoming inaccurate.
+   *
+   * There is a global Geode property, DELTAS_RECALCULATE_SIZE, which can be used to cause all
```

Comment:
I think it might be confusing the calls this a "Geode property" because it might get confused with the gemfire properties. This is actually a java system property that Geode reads. And the actual property is "gemfire.DELTAS_RECALCULATE_SIZE".

Reply:
New version of comment up.